src/lib/JSONViewer.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import {isArray, isObject, isNumber, isString, isBoolean} from 'lodash';
 import {CollapseIcon, isNodeCollapsed, toggleNodeCollapsed} from './CollapseIcon';
+import { viewerStyles } from "./helpers/viewerStyles"
+import { getViewerKey } from "./helpers/ViewerKey"
+import { LabelAndValue } from './components/LabelAndValue'
+import { ViewerLabel } from './components/ViewerLabel'
```

Comment:
why is this renamed?

Reply:
There are two components called Label so I renamed the one associated to the viewerFile to be viewer label